crates/bevy_core_pipeline/src/tonemapping/mod.rs

```diff
@@ -36,12 +46,32 @@ impl Plugin for TonemappingPlugin {
             Shader::from_wgsl
         );
 
+        let mut images = app.world.resource_mut::<Assets<Image>>();
+
+        let tonemapping_luts = TonemappingLuts {
+            blender_filmic: images.add(setup_tonemapping_lut_image(
+                include_bytes!("luts/Blender_-11_12.ktx2"),
+                ImageType::Extension("ktx2"),
+            )),
+            agx: images.add(setup_tonemapping_lut_image(
+                include_bytes!("luts/AgX-default_contrast.ktx2"),
+                ImageType::Extension("ktx2"),
+            )),
+            tony_mc_mapface: images.add(setup_tonemapping_lut_image(
+                include_bytes!("luts/tony_mc_mapface.ktx2"),
+                ImageType::Extension("ktx2"),
+            )),
```

Comment:
Does this mean bevy binaries will grow by > 600kb? If so, would it make sense to add cargo features so that you don't need to include the ones you don't need?

Reply:
I think you are correct, it would increase binary size. For it to be user selectable, this would require 3 separate features correct?

Reply:
Hmm.. Do we have a precedent for adding many related features like this? Maybe a single feature that either embeds all luts or requires the user to add the resource themselves? (e.g. loading it from the network for a web app)

Reply:
Image format support comes to mind, though this still feels more granular. A singe feature may make more sense. Since this resource only lives on the render app I guess it should exist on the main app so a user could access it more reasonably.

Reply:
@coreh Ok, made this a feature now. And if the user inserts their own TonemappingLuts resource before loading the plugin it will use that instead.